src/routes/+page.server.js

```diff
@@ -1,12 +1,23 @@
 /** @type {import('./$types').PageServerLoad} */
 import getSiteConfig from '$lib/server/site'
 // import getNavbarConfig from '$lib/server/navbar'
+import Post from '$lib/server/post'
 
 export async function load ({ fetch }) {
   const site = await getSiteConfig()
   // console.log(
   //   `\nin +page.server.js/getSiteConfig, config: ${JSON.stringify(site)}\n`
   // )
 
-  return { site }
+  // Fetch the posts
+  let posts
+  try {
+    let result = await Post.getAllPosts()
```

Comment:
As I mention above, `Post.getAllPosts()` is extremely wasteful in that it retrieves all posts which need to be filtered after retrieval.